include/rocksdb/utilities/write_batch_with_index.h

```diff
@@ -153,9 +153,9 @@ class WriteBatchWithIndex : public WriteBatchBase {
   // time.
   //
   // The returned iterator should be deleted by the caller.
-  WBWIIterator* NewIterator(ColumnFamilyHandle* column_family);
+  WBWIIterator* NewIterator(ColumnFamilyHandle* column_family, const ReadOptions* read_options = nullptr);
   // Create an iterator of the default column family.
-  WBWIIterator* NewIterator();
+  WBWIIterator* NewIterator(const ReadOptions* read_options = nullptr);
```

Comment:
I am not certain on the coding rules or conventions, but does this need to be marked as "explicit"?

Reply:
It's not an actual constructor nor user-defined conversion function, so no.